### PR TITLE
Update manifest schema to 1.2

### PR DESCRIPTION
**Pull request by billbliss**

### CODE REVIEW

File: msteams-platform/resources/schema/manifest-schema-migrate.md [hunk 1]
```diff
@@ -1,22 +1,22 @@
...
+| `tabs` | `configurableTabs` | In v0.4, all `tabs` were channel/configurable tabs. In v1.x, this distinction is called out to distinguish from `staticTabs` |
+| `tabs:id` | `id` | This field was deprecated in v1.x. Your GUID should migrate to the overall app `id` in v1.x. |
 | `tabs:name` | `name:short`, `name:long` | Use your existing name. |
 | `tabs:description` | `description` | Use your existing tab description or create new ones to represent your entire app experience. |
 | `tabs:description` | `description` | Use your existing tab description or create new ones to represent your entire app experience. |
-| `tabs:icons` | `icons` | Icons have been promoted to a top-level object. Note that [icons have been changed in v1.0](~/publishing/apps-package#icons) and must be included as part of the package. URLs to hosted icons are not allowed. |
+| `tabs:icons` | `icons` | Icons have been promoted to a top-level object.  Note that [icons have been changed in v1.x](~/publishing/apps-package#icons), and must be included as part of the package. URLs to hosted icons are not allowed. |
```
Comment (NormSohl) [inline msteams-platform/resources/schema/manifest-schema-migrate.md]: Extra space added before second sentence. Not needed?

File: msteams-platform/resources/schema/manifest-schema.md [hunk 2]
```diff
@@ -131,16 +131,22 @@ The following schema sample shows all extensibility options.
...
-## ManifestVersion 
+> [!TIP]
+> Specify the schema at the beginning of your manifest to enable IntelliSense or similar support from your code editor: `"$schema": "https://statics.teams.microsoft.com/sdk/v1.2/manifest/MicrosoftTeams.schema.json",`
+
+## manifestVersion 
 
 **Required** &ndash; String
 
-The version of the manifest schema this manifest is using. Should be "1.0".
+The version of the manifest schema this manifest is using.  Should be "1.2".
```
Comment (NormSohl) [inline msteams-platform/resources/schema/manifest-schema.md]: Extra space

**Comment by billbliss** [comment]
Recommended changes completed and committed to PR branch.

**Comment by charlesprakash** [comment]
### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[msteams-platform/resources/schema/manifest-schema-migrate.md](https://github.com/MicrosoftDocs/msteams-docs/blob/manifest-schema-v1.2/msteams-platform/resources/schema/manifest-schema-migrate.md) | :warning:Warning | | [Details](#user-content-msteams-platform%2Fresources%2Fschema%2Fmanifest-schema-migrate.md)
[msteams-platform/resources/schema/manifest-schema.md](https://github.com/MicrosoftDocs/msteams-docs/blob/manifest-schema-v1.2/msteams-platform/resources/schema/manifest-schema.md) | :warning:Warning | | [Details](#user-content-msteams-platform%2Fresources%2Fschema%2Fmanifest-schema.md)

<a name="msteams-platform%2Fresources%2Fschema%2Fmanifest-schema-migrate.md"></a>
### [msteams-platform/resources/schema/manifest-schema-migrate.md](https://github.com/MicrosoftDocs/msteams-docs/blob/manifest-schema-v1.2/msteams-platform/resources/schema/manifest-schema-migrate.md)
 - **Line 9**: **[Warning]** Invalid file link:(~/resources/schema/manifest-schema-v04).
 - **Line 19**: **[Warning]** Invalid file link:(~/publishing/apps-package#icons).

<a name="msteams-platform%2Fresources%2Fschema%2Fmanifest-schema.md"></a>
### [msteams-platform/resources/schema/manifest-schema.md](https://github.com/MicrosoftDocs/msteams-docs/blob/manifest-schema-v1.2/msteams-platform/resources/schema/manifest-schema.md)
 - **Line 10**: **[Warning]** Invalid file link:(~/resources/schema/manifest-schema-migrate).
 - **Line 165**: **[Warning]** Invalid file link:(~/concepts/bots/bots-create).
 - **Line 217**: **[Warning]** Invalid file link:(~/publishing/apps-package#icons).
 - **Line 279**: **[Warning]** Invalid file link:(~/resources/general/developer-preview).
 - **Line 281**: **[Warning]** Invalid file link:(~/concepts/bots/bots-menus).
 - **Line 292**: **[Warning]** Invalid file link:(~/resources/general/developer-preview).
 - **Line 307**: **[Warning]** Invalid file link:(~/resources/general/developer-preview).

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C12%5C12%5Cd98cc23d-94e8-765b-0855-bdd8964dc537%5CPullRequest%5C201712121945379308-20%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=XmRtTtZKj%2FrlMLyLgT83ePogYcHe2R1uUm%2BukRzMxAg%3D&st=2017-12-12T19%3A41%3A25Z&se=2018-01-12T19%3A46%3A25Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**